### PR TITLE
fix(config): add missing Sunricher device configs

### PR DESCRIPTION
**Pull request by taakberg**
This PR adds the following Sunricher devices:
* `ZV9001K4-DIM` Z-Wave Dim Remote Control
* `ZV9001K4-DIM-G2` 2 group single color wall mounted remote
* `ZV9001K12-DIM-Z5` 5 Channel Dimmer Remote Control

It also updates the existing config file for the `ZV9001K12-DIM-Z4` 4 Channel Dimmer Remote Control, adding some missing information and making it more in line with other Sunricher device configs.

### CODE REVIEW

File: packages/config/config/devices/0x0330/zv9001k4-dim.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			"label": "Factory Reset",
+			"description": "Factory reset",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 21930,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0330/zv9001k4-dim.json]: In other devices that have a factory reset parameter, we set `allowManualEntry` to `false` and provide two options - one for the default state and one for the value to factory reset. Please do the same here (and in the other file above).

File: packages/config/config/devices/0x0330/zv9001k12-dim-z4.json [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
+		},
+		"4": {
+			"label": "Launch 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		}
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0330/zv9001k12-dim-z4.json]: These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.

File: packages/config/config/devices/0x0330/zv9001k12-dim-z5.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Launch 5",
+			"maxNodes": 5
+		}
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0330/zv9001k12-dim-z5.json]: These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.

File: packages/config/config/devices/0x0330/zv9001k4-dim-g2.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		}
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0330/zv9001k4-dim-g2.json]: These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.

File: packages/config/config/devices/0x0330/zv9001k4-dim.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		}
+	},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0330/zv9001k4-dim.json]: These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0330/zv9001k4-dim.json (hunk 1 above)
Comment (taakberg) [reply packages/config/config/devices/0x0330/zv9001k4-dim.json]: I've looked trough the manufacturer documentation, and it actually doesn't state any factory reset parameter. I've based this on the [Z-Wave Alliance docs](https://products.z-wavealliance.org/products/3783?selectedFrequencyId=1) of the SR-ZV9001K4-DIM-G2, but I think the information there is incorrect. So I've just opted to remove it instead.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**